### DISCUSSION TITLE
请访问官方中文论坛! https://forum-zh.obsidian.md/

### DISCUSSION THREAD

**Discussion by makerjackie** [obsidianzh/forum#1] [General综合]
2021-9-12: 请访问官方中文论坛! https://forum-zh.obsidian.md/ 本论坛不再维护

## 👋 欢迎！
  我们使用 Github Discussions 作为中文社区的论坛，我们希望你
* 在这里提问
* 分享你的想法
* 融入社区讨论之中
* **友好开放**地对待社区成员，因为这是我们一起构建的社区！

在评论中介绍一下你自己，以及你想在社区中做些什么吧！

📢 欢迎在Q群、微信、知乎、微博等渠道分享这个中文社区！大胆地在这里提问，留言，写下你的建议。

* Q群：774176839，不过由于人数限制，这个群会定时清退发言较少的人。
* [discord中文频道](https://discord.gg/veuWUTm) 部分地区需要科学上网

🔗 Bug / Issue 或者 功能建议 请在 [issue](https://github.com/obsidianzh/forum/issues) 中提出，当然如果你懂英文的话，更建议你在[官方英文论坛](https://forum.obsidian.md/)中提出。（毕竟这是个民间中文社区）其他内容都可以写在 Discussions 中

➡️ **为什么有这个社区？**

很简单，因为不是所有使用者都会英文，我们强烈建议优先看英文文档/论坛，因为更全更新

  ✏️ **社区计划**

- [ ] 在Obsidian 1.0 版本发布时，将[英文帮助文档](https://github.com/obsidianmd/obsidian-translations)翻译为中文。

**Comment by cyddgh** [comment]
更正一点，QQ群的清理是按最后发言的时间，而不是发言条数（或说是QQ软件里的积分）。
